activemodel/lib/active_model/dirty.rb

```diff
@@ -174,12 +174,10 @@ def changed_attributes
     end
 
     # Handles <tt>*_changed?</tt> for +method_missing+.
-    def attribute_changed?(attr, options = nil) #:nodoc:
+    def attribute_changed?(attr, from: :none, to: :none)
       result = changes_include?(attr)
-      if options
-        result &&= options[:to] == __send__(attr) if options.key?(:to)
-        result &&= options[:from] == changed_attributes[attr] if options.key?(:from)
-      end
+      result &&= to == __send__(attr) unless to == :none
+      result &&= from == changed_attributes[attr] unless from == :none
```

Comment:
Hard to imagine the case where attribute value is a symbol. Definitely not the case of AR.